### PR TITLE
add readme n rubric

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # Project 3
+
+For Project 3, you will work independently to create
+an application that solves a problem of your choosing.
+
+## Soft Requirements
```
Comment (jrhorn424) [inline README.md:6]: What are "soft" requirements?

File: rubric.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Rubric
```
Comment (jrhorn424) [inline rubric.md:1]: Would you mind commenting on whether or not anything changed from the previous rubric we used?

Comment (jshawl) [inline rubric.md:1]: deleted presentation category

File: README.md (hunk 1 above)
Comment (adambray) [inline README.md:6]: Yeah, maybe just 'requirements'? And change the wording in the FAQ portion accordingly...

File: rubric.md (hunk 2 above)
Comment (adambray) [inline rubric.md:1]: We should format this so it's clear each line means:
Doesn't meet expectations
Meets expectations
Exceed expectations

**Comment by adambray** [comment]
Other than my notes, this looks good!
:+1:

**Pull request merged**